pkg/controller/aci_istio.go

```diff
@@ -0,0 +1,294 @@
+// Copyright 2020 Cisco Systems, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package controller
+
+import (
+	//"encoding/base64"
+	"fmt"
+	log "github.com/Sirupsen/logrus"
+	istiooperators "github.com/noironetworks/aci-containers/pkg/istiocrd/apis/aci.istio/v1"
+	istiooperatorclientset "github.com/noironetworks/aci-containers/pkg/istiocrd/clientset/versioned"
+	"io/ioutil"
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	restclient "k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
+	"k8s.io/apimachinery/pkg/watch"
+	"os"
+	"os/exec"
+	"time"
+)
+
+// Controller here defines the Operator code handler which list watch the AciIstioOperator
+// object and apply istio-deployment.yaml in the cluster after creation/updation
+type AciIstioController struct {
```

Comment:
nit: ACI should be spelled in caps as per golang convention. We should also do the switch in our other packages, but maybe we could start with istio?

Reply:
Sure, I will rename with my next commit.